enclave-runtime/Cargo.toml

```diff
@@ -104,6 +104,9 @@ sp-std = { version = "4.0.0-dev", default-features = false, git = "https://githu
 sp-utils = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master" }
 sp-version = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master" }
 sp-application-crypto = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master" }
+#beefy-merkle-tree = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master", features = "keccak" }
```

Comment:
That's the ugly part here: substrate actually has a no-std compatible merkle tree primitives crate: https://github.com/paritytech/substrate/tree/master/frame/beefy-mmr/primitives

However, this was introduced on 23rd of September. We are not yet on this substrate commit. Can you agree with this ugly solution and remove it once we are on this commit? If yes, I'll create an appropriate issue ofc.

Reply:
If it's well documented and even has a corresponding issue, that's totally fine for me 👍

Reply:
I think it's fine too. 👍